contracts/epochs/FlowEpoch.cdc

```diff
@@ -147,6 +147,134 @@ access(all) contract FlowEpoch {
         dkgPubKeys: [String]
     )
 
+    /// The Epoch Recover service event is emitted when a recoverEpoch transaction is submitted to the network. When the network 
+    /// gets into an EFM state the recoverEpoch transaction is used to override the current epoch and recover the network without 
+    /// sporking. This service event is processed by protocol participants to update their local protocol state with the new recover
+    /// epoch data.
+    access(all) event EpochRecover (
```

Comment:
That's a massive event. Are you sure it's OK to include so much data right into the event?

It also seems like this event has (almost?) the same contents as `RecoverEpochMetadata`?